src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -326,6 +326,38 @@ void IslandWindow::OnSize(const UINT width, const UINT height)
         _windowCloseButtonClickedHandler();
         return 0;
     }
+    case WM_MOUSEWHEEL:

```

Comment:
so this doesn't break with surface devices, but _does it change with a wired mouse with a wheel?_

Reply:
I can't explain it, but yea this isn't hit for a normal mouse wheel either. Maybe this is what `MouseInPointer` does? Maybe when XAML _does_ handle it, it handles it via the `WM_POINTERWHEEL`, but then if it doesn't, it arrives to us as a `WM_MOUSEWHEEL`? Maybe that's the reason these devices are broken in the first place? I have no idea. 

Regardless, this is working as expected.